owa/app/js/subscription/subscription.controller.js

```diff
@@ -48,6 +51,23 @@ class SubscriptionController {
         vm.subscription = response.results[0];
       })
     }
+
+    //Check if collection version has been passed to subscription url by checking number of forward slashes
```

Comment:
Can you give an example of what that would look like?

Reply:
In the comments or in the code?

url without version (4 forward slashes)
https://api.openconceptlab.org/users/username/collections/collectionname

url with version (5 forward slashes)
https://api.openconceptlab.org/users/username/collections/collectionname/v1.0

5 forward slashes means you have provided the collection version and hence you can't subscribe to a snapshot

Reply:
Which one do you think is better?

Reply:
@dkayiwa  I'm still not sure about this, does it make sense to have a lot of comments in the code?

Reply:
How much is a lot?

Reply:
I came up with
    /*
    *This checks if collection version has been passed to subscription url by checking number of forward slashes
    *If the number is 5, such as with https://api.openconceptlab.org/users/username/collections/collectionname/v1.0
    *that means collection version was passed and isVersionAdded() will return true
    */

hopefully it's good enough?